examples/multiphase_flow/ex11/LSLocateStructureInterface.cpp

```diff
@@ -166,7 +166,10 @@ LSLocateStructureInterface::getExtremeCoords(std::vector<IBTK::Vector>& corners,
     double xmax = -std::numeric_limits<double>::max();
     double ymin = std::numeric_limits<double>::max();
     double ymax = -std::numeric_limits<double>::max();
-    double y_xmin, y_xmax, x_ymin, x_ymax;
+    double y_xmin = std::numeric_limits<double>::quiet_NaN();
+    double y_xmax = std::numeric_limits<double>::quiet_NaN();
+    double x_ymin = std::numeric_limits<double>::quiet_NaN();
+    double x_ymax = std::numeric_limits<double>::quiet_NaN();
 
```

Comment:
@nishantnangia : Are these new default values OK?

Reply:
These should be fine.